broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -47,53 +37,44 @@ async function commit ({ params, relayer, logger, engines, orderbooks }, { Empty
+    engine.getPaymentChannelNetworkAddress(),
+    inverseEngine.getPaymentChannelNetworkAddress()
+  ])
+
+  logger.info(`Attempting to create channel with ${relayerAddress} on ${symbol} with ${balanceCommon}`, {
     balanceCommon,
     balance
   })
 
-  // We use common units for these calculation so that we can provide
-  // friendly errors to the user.
-  // TODO: Get correct fee amount from engine
-  if (MINIMUM_FUNDING_AMOUNT.gt(balanceCommon)) {
-    throw new Error(`Minimum balance of ${MINIMUM_FUNDING_AMOUNT} needed to commit to the relayer`)
-  } else if (maxChannelBalance.lt(balance)) {
-    const maxChannelBalanceCommon = Big(maxChannelBalance).div(engine.quantumsPerCommon).toString()
-    logger.error(`Balance from the client exceeds maximum balance allowed (${maxChannelBalance.toString()}).`, { balance })
-    throw new Error(`Maximum balance of ${maxChannelBalanceCommon} ${symbol} exceeded for ` +
-      `committing of ${balanceCommon} ${symbol} to the Relayer. Please try again.`)
-  }
-
-  // Get the max balance for outbound channel to see if there are already channels with the balance open. If this is the
-  // case we do not need to go to the trouble of opening a new channel
-  const { maxBalance: maxOutboundBalance } = await engine.getMaxChannel()
-
-  // If maxOutboundBalance exists, we need to check if the balance is greater or less than the balance of the channel
-  // we are trying to open. If maxOutboundBalance does not exist, it means there are no channels open and we can safely
-  // attempt to create channels with the balance
-  if (maxOutboundBalance) {
-    const insufficientOutboundBalance = maxOutboundBalance && Big(maxOutboundBalance).plus(engine.feeEstimate).lt(balance)
-
-    if (insufficientOutboundBalance) {
-      logger.error('Existing outbound channel of insufficient size', { desiredBalance: balance, feeEstimate: engine.feeEstimate, existingBalance: maxOutboundBalance })
-      throw new Error('You have an existing outbound channel with a balance lower than desired, release that channel and try again.')
-    }
-  }
-
-  if (!maxOutboundBalance) {
-    logger.debug('Creating outbound channel', { address, balance })
+  const currentBalance = await engine.getTotalBalanceForAddress(relayerAddress)
+  let balanceToCommit = Big(balance).minus(currentBalance)
 
+  if (balanceToCommit.lte(0)) {
+    logger.debug('Channels with sufficient balance already exist', { balance: currentBalance })
+  } else {
     try {
-      await engine.createChannel(address, balance)
+      logger.info(`Opening outbound channels`, { relayerAddress, symbol, balanceToCommit: balanceToCommit.toString() })
+      await engine.createChannels(relayerAddress, balanceToCommit.toString())
     } catch (e) {
       logger.error('Received error when creating outbound channel', { error: e.stack })
-      throw new Error(`Funding error: ${e.message}`)
+
+      if (e && e.message && e.message.includes('too small') && Big(currentBalance).gt('0')) {
```

Comment:
[nit] just curious, is there anything special about using a `'0'` vs `0`. I would make this consistent with the code above by using a number